packages/postcss-merge-rules/src/lib/ensureCompatibility.js

```diff
@@ -55,6 +56,47 @@ export const pseudoElements = {
   '::marker': 'css-marker-pseudo',
   '::placeholder': 'css-placeholder',
   '::selection': 'css-selection',
+  // FROM MDN
+  ':any-link': 'any-link',
+  ':blank': 'blank',
+  ':current': 'css-current',
+  ':defined': 'css-defined',
+  ':dir()': 'css-dir',
+  ':drop': 'css-drop',
+  ':first': 'css-first',
+  ':fullscreen': 'css-fullscreen',
+  ':future': 'css-future',
+  ':host()': 'host()',
+  ':host-context()': 'host-context()',
+  ':is()': 'css-is',
+  ':lang()': 'css-lang',
+  ':not()': 'css-not',
+  ':nth-child()': 'css-nth-child',
+  ':nth-col()': 'css-nth-col',
+  ':nth-last-child()': 'nth-last-child',
+  ':nth-last-of-type()': 'nth-last-of-type',
+  ':nth-of-type()': 'nth-of-type',
+  ':past': 'css-past',
+  ':required': 'css-required',
+  ':right': 'right',
+  ':scope': 'css-scope',
+  ':target-within': 'css-target-within',
+  ':user-invalid': 'css-user-invalid',
+  // ':valid': 'css-valid',
+  // ':visited': 'css-visited',
+  ':where()': 'css-where',
+
+  // pseudo-elements
+  '::after (:after)': '::after',
+
+  '::before (:before)': '::before',
+  '::first-letter (:first-letter)': 'first-letter',
+  '::first-line (:first-line)': 'first-line',
+  '::grammar-error': 'grammar-error',
+
+  '::part()': '::part',
+  '::slotted()': '::slotted',
```

Comment:
Why some have `()` parenthesis, some doesn't have? It is bug?

Reply:
let me check

Reply:
Ok these should not have `()` but these are `level 4 CSS Selector` except for the `visited`.

And I think level 4 is not supported by `caniuse-lite` , so this all will be removed